plugins/modules/splunk_data_inputs_monitors.py

```diff
@@ -0,0 +1 @@
+/home/pranav/Desktop/RedHat/collections/ansible_collections/splunk/es/plugins/modules/data_inputs_monitors.py
```

Comment:
@pranav-bhatt this is resulting into failure of tests, u have to mention just the `data_inputs_monitors.py` for redirection